cards.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
```

Comment:
Perfect! 💯  just make sure to include comments for each of your methods (functionality / what it returns)